src/index.vue

```diff
@@ -104,6 +81,38 @@ export default {
             type: String,
             default: "hover"
         }
+    },
+    data() {
+        return {
+            currentCategoryMenu: ""
+        }
+    },
+    watch: {
+        menuLinks: {
+            handler(links) {
+                this.currentCategoryMenu = links.length ? links.find(link => link.menuId).menuId : "";
+            },
+            immediate: true
+        }
+    },
+    computed: {
+        menuLinks() {
+            return this.resources.map( item => {
+                if (item.links) {
+                    item.menuId = uuid();
```

Comment:
I think this should be coming from backend. You can ask them to include the link or menu ID as needed.